web/src/lib/common/utils.ts

```diff
@@ -97,3 +97,13 @@ export function padFloatToDecimalPlaces(num: number, decimalPlaces: number) {
 	const factor = Math.pow(10, decimalPlaces);
 	return (Math.round(num * factor) / factor).toFixed(decimalPlaces);
 }
+
+export function classToPOJO<T extends object>(instance: T): { [K in keyof T]: T[K] } {
```

Comment:
Just in case you need a more robust solution:

https://www.npmjs.com/package/class-transformer

Although hmm it's been 2 years since their last push but they still have a consistent 3M weekly downloads.

Reply:
yeah I see...  thanks for bringing that up, I just found an easier built-in way.

```
export function toPlainObject<T extends object>(instance: T) {
	return Object.assign({}, instance);
}
```

I think the task is trivial enough to not require another external library.